src/internal/direction.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export function isRtl(element: Element) {
+  let isRtl = false;
+
+  if (element instanceof Element) {
```

Comment:
This `instanceof` check is not really necessary if the type is already declared as `Element` in the function definition, right?

Reply:
I need to leave it because I am temporarily setting the type back to `HTMLElement | EventTarget`. Also, there were some tests that broken when I didn't run this check. We can finalize this in the follow up PR for popover.